pages/docs/integrations/grouparoo-bigquery.mdx

```diff
@@ -1,24 +1,229 @@
 ---
-title: "@grouparoo / bigquery"
-date: "2021-10-12"
+title: "BigQuery"
+date: "2022-01-20"
 pullQuote: "Learn how to work with the Google BigQuery Plugin as a Source."
 pageNavSelector: "h2"
 ---
 
 <PluginDocsLinks plugin="BigQuery" />
 
-Grouparoo's [BigQuery](https://cloud.google.com/bigquery/) Plugin enables you to import data from one or more BigQuery databases.
+## Overview
 
-This guide will show you how to work with the BigQuery Plugin to [create a Source](#create-bigquery-source) to import your data.
+[BigQuery](https://cloud.google.com/bigquery) is a Google-run serverless, scalable, cloud-based data warehouse. It is built to support analysis and has built-in machine learning capabilities. Specifically, BigQuery is designed to help companies look for insights using very large datasets. Grouparoo's BigQuery Plugin enables you to import data from one or more BigQuery databases.You can use Grouparoo to sync data from BigQuery to any of the Destinations we provide plugins for.
 
-<PluginDocsInstallation plugin="BigQuery" />
+### Sources
 
-<PluginDocsCreateApp plugin="BigQuery" />
+<table className="docTable">
+  <tablebody>
+    <tr>
+      <td>Source Name</td>
+      <td>Description</td>
+    </tr>
+    <tr>
+      <td>
+        <a href="#bigquery-import-table">
+          <strong>BigQuery Import Table</strong>
+        </a>
+      </td>
+      <td>Import or update Records from a Bigquery database table.</td>
+    </tr>
+    <tr>
+      <td>
+        <a href="#bigquery-import-query">
+          <strong>BigQuery Import Query</strong>
+        </a>
+      </td>
+      <td>Import or update records via a custom Bigquery query.</td>
+    </tr>
+  </tablebody>
+</table>
 
-<ValidateAndApplyConfig headingLevel="h3" />
+## Setup
 
-<PluginDocsColumnarSource plugin="BigQuery" />
+In order to use BigQuery, you’ll need to enable it as a service in your Google Cloud project. Create a service account which will be the source for a Client Email Address and Private key you will need to configure the initial connection for a BigQuery App in Grouparoo.
 
```

Comment:
screenshot?

Reply:
Working on this. There will probably be two for way-finding.